expenser-ui/src/app/create-dialog/investment-form.tsx

```
@@ -190,12 +195,37 @@ export function InvestmentForm({ initialData, onSubmit }: InvestmentFormProps) {
+                    const rawValue = e.target.value.replace(/[^0-9.]/g, "");
+                    field.onChange(rawValue);
+
+                    if (typingTimeout) {
+                      clearTimeout(typingTimeout);
+                    }
+
+                    setTypingTimeout(
+                      setTimeout(() => {
+                        if (rawValue) {
+                          const numericValue = parseFloat(rawValue);
+                          if (!isNaN(numericValue)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)